artifactory/artifacts_test.go

```diff
@@ -101,7 +102,11 @@ func Test_Artifacts(t *testing.T) {
 				g.Assert(actual == nil).IsTrue()
 				g.Assert(resp == nil).IsTrue()
 				g.Assert(err != nil).IsTrue()
-				g.Assert(err.Error()).Equal("open this/is/a/bad/path.txt: no such file or directory")
+				if runtime.GOOS == "windows" {
+					g.Assert(err.Error()).Equal("open this/is/a/bad/path.txt: The system cannot find the path specified.")
+				} else {
+					g.Assert(err.Error()).Equal("open this/is/a/bad/path.txt: no such file or directory")
```

Comment:
I'd propose we update the error checking to be a bit smarter instead of just doing keyword matching. For example:

```suggestion
					g.Assert(err).Equal(&os.PathError{Op:"open", Path:"this/is/a/bad/path.txt", Err:unix.ENOENT})
```

Reply:
Much better. Thanks.

Reply:
Actually, might it be better to do the following instead?

`g.Assert(os.IsNotExist(err)).IsTrue()`

Reply:
I'm good with that syntax if it works across platforms.